_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
+  variants: [ us ]
+
+-
+  code: he
+  name: Hebrew
+  family: sem
+  scripts: [ Hebr ]
+  typology:
+    - VSO
+    - analytic, inflected
+  variants: [ il ]
+
+-
+  code: hi
+  name: Hindi
+  family: inc
+  scripts: [ Deva, Latn, Mahj ]
+  typology:
+    - SOV
+  variants: [ in ]
+
+-
+  code: hmn
+  name:  	Hmong
+  family: hmx
+  scripts: [ Latn, Hmng ]
+  typology:
+    - SVO
+    - analytic
+  variants:
+
+-
+  code: hr
+  name: Croatian
+  family: sla
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected, fusional
+  variants: [ hr, ba ]
+
+-
+  code: ht
+  name: Haitian
+  family: hat
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ ht ]
+
+-
+  code: hu
+  name: Hungarian
+  family: fiu
+  scripts: [ Latn ]
+  typology:
+    - SOV
+    - agglutinative, synthetic
+  variants: [ hu ]
+
+-
+  code: id
+  name: Indonesian
+  family: mal
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ id ]
+
+-
+  code: ig
+  name: Igbo
+  family: ibo
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ ng ]
+
+-
+  code: is
+  name: Icelandic
+  family: ice
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ is ]
+
+-
+  code: it
+  name: Italian
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - fusional, synthetic
+  variants: [ ch, it ]
+
+-
+  code: ja
+  name: Japanese
+  family: jpn
+  scripts: [ Jpan ]
+  typology:
+    - SOV
+    - agglutinative, synthetic
+  variants: [ jp ]
+
+-
+  code: jv
+  name: Javanese
+  family: map
+  scripts: [ Java, Latn ]
+  typology:
+    - SVO
+    - agglutinative
+  variants: [ id ]
+
+-
+  code: ka
+  name: Georgian
+  family: cau
+  scripts: [ Geor ]
+  typology:
+    - SVO
+    - agglutinative, synthetic
+  variants: [ ge ]
+
+-
+  code: kk
+  name: Kazakh
+  family: trk
+  scripts: [ Cyrl, Arab, Latn ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ kz ]
+
+-
+  code: km
+  name: Khmer
+  family: aav
+  scripts: [ Khmr ]
+  typology:
+    - SVO
+    - analytic, isolating
+  variants: [ kh ]
+
+-
+  code: kn
+  name: Kannada
+  family: dra
+  scripts: [ Knda ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ id ]
+
+-
+  code: ko
+  name: Korean
+  family: kor
+  scripts: [ Kore ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ kr ]
+
+-
+  code: ku
+  name: Kurdish
+  family: kur
+  scripts: [ 	Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - inflected
+  variants: [ iq ]
+
+-
+  code: ky
+  name: Kyrgyz
+  family: trk
+  scripts: [ 	Arab, Cyrl, Latn ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ kg ]
+
+-
+  code: la
+  name: Latin
+  family: lat
+  scripts: [ Latn ]
+  typology:
+    - SOV, SVO
+    - synthetic, fusional
+  variants:
+
+-
+  code: lb
+  name: Luxembourgish
+  family: gem
+  scripts: [ Latn ]
+  typology:
+    - V2 SOV
+    - synthetic
+  variants: [ lu ]
+
+-
+  code: lo
+  name: Lao
+  family: lao, tha
```

Comment:
unfortunately its family has no code

https://en.m.wikipedia.org/wiki/Kra–Dai_languages

So just empty or null

Reply:
We can use `tai`.